sample-bchd.conf

```diff
@@ -217,14 +236,23 @@
 ; All ipv6 interfaces on non-standard port 8337:
 ;   rpclisten=[::]:8337
 
+; File containing the certificate file
+; rpccert=~/.bchd/rpc.cert
```

Comment:
Is this a good default location for the cert and key?

Reply:
yup